README.md

```diff
@@ -1,2 +1,106 @@
-# covid-19-world-wide
-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] Kindly please make a video for this project., according to and following closely these requirements:


![image](https://user-images.githubusercontent.com/57421823/152125226-e758e5f8-0d7c-4f5a-b5c4-aede8baf912c.png)

- [ ] Let's give credit to this design's author! According to these instructions:


![image](https://user-images.githubusercontent.com/57421823/152125684-6fa7d8fd-c6a0-4bea-9fcb-ede03127b059.png)


- [ ] You need to please improve your readme.   Please kindly follow this [template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

That way, your app will be well documented 👍